README.md

```diff
@@ -63,12 +64,4 @@ This will create a new branch. Now run:
 ./scripts/before_prod_deploy.sh --l10n-repo .
 ```
 
-Git commit the changes made by the script. Pull request that branch against `master`.
-
-After that pull request is merged run:
-
-```
-./scripts/email-dev-l10n.sh
```

Comment:
We have decommissioned mailing list years ago, so this is not useful (also removed the script).